pkg/asset/agent/image/unconfigured_ignition.go

```diff
@@ -249,8 +222,6 @@ func (a *UnconfiguredIgnition) Generate(_ context.Context, dependencies asset.Pa
 		return err
 	}
 
-	addMirrorData(&config, registriesConfig, registryCABundle)
```

Comment:
Why is this being removed? It wasn't added by the interactive disconnected workflow.